packages/frontend/core/src/modules/open-in-app/views/open-in-app-page.tsx

```diff
@@ -41,7 +41,7 @@ export const OpenInAppPage = ({
   const goToAppearanceSetting = useCallback(
     (e: MouseEvent) => {
       openHereClicked?.(e);
-      workspaceDialogService.open('setting', {
+      workspaceDialogService?.open('setting', {
```

Comment:
it's not kinda proper to show [change in settings] in authentication pages

Reply:
It will not show on authentication mode. The button will only show when mode === 'open-doc'.